examples/lock-app/lock-common/lock-app.zap

```diff
@@ -2126,6 +2126,14 @@
               "source": "client",
               "isIncoming": 1,
               "isEnabled": 1
+            },
+            {
+              "name": "RetrieveLogsResponse",
```

Comment:
Is this the correct thing to do? Do we have to enable every `response`?
It looks odd in .matter file to say `server handles the command FooResponse` overall however I see some existing examples doing this.

I do not quite understand this bit ... I expect servers to need to handle Requests but never Responses (they send these out only). what is the expected behaviour?

Reply:
Diagnostic logs is optional cluster but if you enable the cluster `RetrieveLogsResponse` is a mandatory command according the spec.